CHANGELOG.md

```diff
@@ -3,7 +3,54 @@
+- ast: Fix regression in `opa check` where a file that referenced non-provided schemas failed validation ([#7124](https://github.com/open-policy-agent/opa/pull/7124)) authored by @tjons
+- test/cases/testdata: Fix bug in test by replacing unification by explicit equality check ([#7093](https://github.com/open-policy-agent/opa/pull/7093)) authored by @matajoh
+- topdown: Use new Inter-Query Value Cache for `json.match_schema` built-in function ([#7011](https://github.com/open-policy-agent/opa/issues/7011)) authored by @anderseknert reported by @lcarva
+
+### Runtime, Tooling, SDK
+- cmd: Make `opa check` respect `--ignore` when `--bundle` flag is set ([#7136](https://github.com/open-policy-agent/opa/issues/7136)) authored by @anderseknert
+- server/writer: Use Header().Set() not Header().Add() to avoid duplicate content-type headers  ([#7114](https://github.com/open-policy-agent/opa/pull/7114)) authored by @srenatus
+- cmd: Support `file://` format for TLS key material file flags in `opa run` ([#7094](https://github.com/open-policy-agent/opa/pull/7094)) authored by @alexrohozneanu
+- plugins/rest/azure: Support managed identity for App Servivce / Container Apps ([#7085](https://github.com/open-policy-agent/opa/issues/7085)) reported and authored by @apc-kamezaki
+- debug: Fix step-over behaviour when exiting partial rules ([#7096](https://github.com/open-policy-agent/opa/pull/7096)) authored by @johanfylling
+- util+plugins: Fix potential memory leaks with explicit timer cancellation ([#7089](https://github.com/open-policy-agent/opa/pull/7089)) authored by @philipaconrad
+
+### Docs, Website, Ecosystem
+- docs: Fix OCI example with updated flag used by the ORAS CLI  ([#7130](https://github.com/open-policy-agent/opa/pull/7130)) authored by @b3n3d17
+- docs: Delete Atom editor from supported editor integrations ([#7111](https://github.com/open-policy-agent/opa/pull/7111)) authored by @KaranbirSingh7
+- docs/website: Add Styra OPA ASP.NET Core SDK integration ([#7073](https://github.com/open-policy-agent/opa/pull/7073)) authored by @philipaconrad
+- docs/website: Update compatibility information on the rego-cpp integration ([#7078](https://github.com/open-policy-agent/opa/pull/7078)) authored by @matajoh
+
+### Miscellaneous
+- Replace use of yaml.v2 library with yaml.v3 ([#7090](https://github.com/open-policy-agent/opa/issues/7090)) authored by @anderseknert
+- Dependency updates; notably:
+  - build(deps): bump aquasecurity/trivy-action from 0.24.0 to 0.28.0 (#7123)
+  - build(deps): bump github.com/containerd/containerd from 1.7.22 to 1.7.23
+  - build(deps): bump github.com/prometheus/client_golang from 1.20.4 to 1.20.5
+  - build(deps): bump golang.org/x/net from 0.29.0 to 0.30.0
+  - build(deps): bump golang.org/x/time from 0.6.0 to 0.7.0
+  - build(deps): bump google.golang.org/grpc from 1.67.0 to 1.67.1
```

Comment:
I think I wouldn't call out any of these. 🤔 Let's definitely drop the trivy-action one, since it's least relevant to the release itself.

Reply:
I've dropped trivy-action. I've kept the other as we've mentioned them in the past.